utils/release-candidate-testing/node/index.js

```diff
@@ -0,0 +1,121 @@
+import { RedisClusterClient } from "@aws/glide-for-redis";
+import { exec } from "child_process";
+
+const PYTHON_CLUSTER_SCRIPT_PATH = "../../../utils/cluster_manager.py";
+const PYTHON = "python3";
+const PYTHON_SCRIPT_RUN = `${PYTHON} ${PYTHON_CLUSTER_SCRIPT_PATH}`;
+const CLUSTER_NODES_AND_REPLICAS_PARAMS = `-r 3 -n 3 --cluster-mode`;
+let clusterFolder = "";
+
+function parseOutput(input) {
```

Comment:
Can you export this function from TestUtilities.ts instead of duplicating it?

Reply:
maybe you can move all of the cluster related code to a separate module that you can use both in tests and elsewhere

Reply:
you're also missing standalone tests